### ISSUE TITLE
Considerable bouncing effect of the progress bar when using the Bootstrap theme

### ISSUE THREAD

**Issue by mauromol** [primefaces/primefaces#909]
When using the Bootstrap theme, the progress bar, when updating, shows an annoying (even ironic, I would say) bouncing effect after upgrading to PF 5.3. It didn't happen in 5.2.

This is clearly visible in the demo, too, after switching to Bootstrap theme: http://www.primefaces.org/showcase/ui/misc/progressBar.xhtml

Is it wanted?

**Comment by cagataycivici** [comment]
I couldn't see it, could you update a video?

**Comment by mauromol** [comment]
Here it is: https://youtu.be/2QuXq9gMYNc
It seems like it happens in Firefox, but not in Chrome. Currently using Firefox 42.0.

**Comment by melloware** [comment]
@mauromol Can you reproduce on the latest showcase?  I am using Firefox Quantum 62.0 and I cannot reproduce your issue.

**Comment by melloware** [comment]
Looks like this must be Bootstrap Theme issue still has it is still happening in the current showcase.

**Comment by mauromol** [comment]
Yes, it's still observable in current showcase.

**Comment by Alister71** [comment]
It is observable in primefaces 6.2 and was not observable in 6.1

**Comment by melloware** [comment]
It looks like this has been a strange issue since PF 5.3

**Comment by Alister71** [comment]
Yes, last check showed that it was pre 6.1 isssue

**Comment by melloware** [comment]
Bootstrap theme CSS tries to ease the animation but makes it worse.

```css
.ui-progressbar .ui-progressbar-value.ui-widget-header {
...
-webkit-transition:width 0.6s ease;
-moz-transition:width 0.6s ease;
-ms-transition:width 0.6s ease;
-o-transition:width 0.6s ease;
transition:width 0.6s ease;
}
```

if you override the CSS with your own CSS to turn off the transition:width it resolves the issue.

**Comment by melloware** [comment]
Fixed with PR: https://github.com/primefaces/themes/pull/14

**Comment by tandraschko** [comment]
Duplicate: https://github.com/primefaces/themes/issues/13 (and more related to themes)

**Comment by mauromol** [comment]
Thanks Melloware and Thomas! What's the target milestone?

**Comment by tandraschko** [comment]
There is currently no milestone.
We will first release the next PF core and then we should check to release the themes again.

**Comment by melloware** [comment]
To fix it now @mauromol just go here and download and do mvn clean install since this was fixed in the 1.0.11 BootStrap theme.  https://github.com/primefaces/themes
